packages/compressed-textures/src/resources/CompressedTextureResource.ts

```diff
@@ -149,6 +149,8 @@ export class CompressedTextureResource extends BlobResource
             return false;
         }
 
+        gl.pixelStorei(gl.UNPACK_ALIGNMENT, 4);
```

Comment:
Is this intentional, or should we use `this.unpackAlignment`?

Reply:
I'm not very familiar with this part code. But the level buffers always have an alignment of 4 (`_createLevelBuffers` is called with `blockWidth/blockHeight` 4). We could change it to `gl.pixelStorei(gl.UNPACK_ALIGNMENT, this.unpackAlignment);`, because the default is 4, but it would probably make more sense if `CompressedLevelBuffer` had an  `levelUnpackAlignment` property as each buffer could (at least in theory) have different alignments. Hardcoding it to 4 definitely doesn't break anything. I wasn't 100% sure what the best solution here would be. I'm open for suggestion.

Reply:
Yeah I think it is fine to set unpack align to 4 for now. We can add more options later if needed.